bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/package-info.java

```diff
@@ -34,7 +34,7 @@
  *      version, is bound to this proxy component resource type.
  * </p>
  */
-@Version("12.20.0")
+@Version("13.0.0")
```

Comment:
No major bumps, this will cause customer bundles to no longer start on an upgrade.